.travis.yml

```diff
@@ -14,6 +14,12 @@ addons:
   postgresql: 9.4
 
 rvm:
+  - 2.3.0
```

Comment:
Indeed, this will make every Travis build go a looooong time,. which means longer delay before I know that a PR can be safely merged. I suggest just 2.3.1 and 2.4.1